### DISCUSSION TITLE
How to make ZEN work (mostly) on Windows versions Vista and up using r3dfox

### DISCUSSION THREAD

**Discussion by sometgirldotonline** [zen-browser/desktop#348] [General]
Hi all, I thought I'd quickly note this, but you can patch Zen to work on Windows Vista and upwards using a browser called r3dfox as a base.
1. Install R3dFox
2. Download a portable version of Zen
3. Copy the "browser" "defaults" "fonts" and "gmp-clearkey" folders over to your redfox install directory and allow replacing
4. Complete

As of current, the NTP is blank, I am unsure if this is by design of Zen (have not tried it on an actually supported OS yet)

also 5 stars to the Zen browser, it seems really nice.

**Comment by sometgirldotonline** [comment]
update: the hamburger menu in top right is also broken

**Comment by jrjakiro** [comment]
Is this development setup works for Mac? Can we do code changes, compile and build it in Mac?

**Comment by sometgirldotonline** [comment]
huh? nowhere in my post do i mention anything about changes and such?
this is a post about making Zen partially work on Win Vista and up.
I'm not a Zen developer, however I think they use the normal firefox build
system.
